src/components/Wheel/index.js

```diff
@@ -1,11 +1,33 @@
 import React, { useRef, useEffect, useState } from 'react';
 
 import ModalWindow from '../ModalWindow';
-import { fortune } from '../../data/index.json';
 import Button from '@material-ui/core/Button';
 import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
 
-function Wheel() {
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { fetchFortuneData } from '../../actions/fortune';
+
+const mapStateToProps = (props) => ({
+  fortune: props.fortune.fortune,
+});
+
+const mapDispatchToProps = (dispatch) =>
+  bindActionCreators(
+    {
+      fetchFortuneData: fetchFortuneData,
+    },
+    dispatch,
+  );
+
+const Wheel = (props) => {
+  const { fortune = [] } = props;
+  useEffect(() => {
+    props.fetchFortuneData();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why do you need to disable eslint here?

Reply:
fixed. update will be in next commit

Reply:
How have you fixed it?
If you can't fix it, tell me a reason.